docker-compose.yml

```diff
@@ -253,6 +255,54 @@ services:
       - _APP_LOGGING_PROVIDER
       - _APP_LOGGING_CONFIG
 
+  appwrite-worker-syncs-out:
+    entrypoint: worker-syncs-out
+    <<: *x-logging
+    container_name: appwrite-worker-syncs-out
+    image: appwrite-dev
+    build:
+      context: .
+    networks:
+      - appwrite
+    volumes:
+      - ./app:/usr/src/code/app
+      - ./src:/usr/src/code/src
+    depends_on:
+      - mariadb
+      - redis
+    environment:
+        - _APP_ENV
+        - _APP_REDIS_HOST
```

Comment:
In my new branch, we have new env vars to support new connections. We should also keep the old `_APP_REDIS` vars for backward compatibility. You can see examples in the docker-compose file in my db-pool branch: https://github.com/appwrite/appwrite/pull/4330